README.md

```diff
@@ -62,14 +48,6 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Good job with your readme file, It looks professional and it is descriptive. But the link to your license is broken :cry:  Please don't forget to add a license to your project and I recommend using the [MIT license](https://choosealicense.com/licenses/mit/), Set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). another resource is [any other license](https://choosealicense.com/licenses/) . :smile: